bom/pom.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright DataStax, Inc.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>com.datastax.oss</groupId>
+    <artifactId>java-driver-parent</artifactId>
+    <version>4.5.0-SNAPSHOT</version>
+  </parent>
+  <artifactId>java-driver-bom</artifactId>
+  <packaging>pom</packaging>
+  <name>DataStax Java driver for Apache Cassandra(R) - Bill Of Materials</name>
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>${project.groupId}</groupId>
+        <artifactId>java-driver-core</artifactId>
+        <version>${project.version}</version>
```

Comment:
The original reporter of the jira ticket advised against that:
> A good practice is to NOT use project.version property when declaring a version, but the real version

But it works, so unless someone can explain why this would be a bad idea, I'm going with the simplest option.

I would even have omitted the version altogether, because we already have a `<dependencyManagement>` section in the parent POM. But apparently this one doesn't inherit from it.

Reply:
@mp911de would you mind taking a look? ^

Reply:
I'm not exactly sure about the full rationale behind why placeholders should not be used in a BOM declaration. From what I know is that placeholders require resolution during consumption. With the current arrangement, a consumer needs to fetch `java-driver-parent` just to be able to resolve version attributes regardless of whether dependencies are included in the project.

A typical project using the BOM directly shouldn't be impacted too much as they would depend on the Cassandra driver anyway. For frameworks (such as Spring Boot), they typically come with a wide dependency management arrangement that points to pre-defined versions while a lot of users do not use all declared dependency versions.

Reply:
The rationale is that `${project.version}` is a bit misleading in the sense of it is a standard property of the project. Right now Maven resolves that token locally to the project (so it uses the version of the bom) but if you look at `${project.version}` in a project that imports the BOM, it does not refer to the same value at all. 

That's why things like the flattened maven plugin resolves the version and hardcodes it to make it more explicit. I'd recommend using that plugin btw, especially for a bom.